umm-spec-lib/src/cmr/umm_spec/test/expected_conversion.clj

```diff
@@ -445,7 +445,10 @@
 (defmethod umm->expected-convert :iso-smap
   [umm-coll _]
   (let [original-brs (get-in umm-coll conversion-util/bounding-rectangles-path)
+        original-platforms (:Platforms umm-coll)
```

Comment:
Looked down too lines below.  For some reason, we need to call (umm->expected-convert umm-coll :iso19115), which set all the platform stuff to nil. but that's for iso19115, it doesn't apply now to the iso-smap with the new implementation. so I need to add the platform info back.